test/integration/members.test.js

```diff
@@ -89,4 +103,38 @@ describe('Members', function () {
         })
     })
   })
+
+  describe('PATCH /members/moveToMembers', function () {
+    before(async function () {
+      await cleanDb()
+      const userId = await addUser(superUser)
+      jwt = authService.generateAuthToken({ userId })
+    })
+    it('Should make the user a member', function (done) {
```

Comment:
We need test cases for all the 2xx and 4xx responses:
1. Requested user not existing
2. User making the request is not authenticated